test/browser/browser.sh

```diff
@@ -57,7 +57,7 @@ systemctl start firewalld
 firewall-cmd --add-service=cockpit --permanent
 firewall-cmd --add-service=cockpit
 
-if grep -Eq 'PLATFORM_ID=.*(f35)' /etc/os-release; then
+if grep -Eq 'PLATFORM_ID=.*(f35|f36|el9)' /etc/os-release; then
```

Comment:
I think we should invert that too, and *not* do it on f34 and el8. But that can also be a follow-up the next time we have to touch this file.